client/gutenberg/editor/style.scss

```diff
@@ -411,10 +411,15 @@ $gutenberg-theme-toggle: #11a0d2;
 }
 
 // Calypsoify iframe
+
+html.is-iframe {
+	overflow: hidden;
+}
+
 .main.calypsoify.is-iframe {
 	background-color: $gray-light;
 	margin: 0;
-	overflow-y: scroll;
+	overflow-y: hidden;
```

Comment:
I think we can remove this property at all. `.main.calypsoify.is-iframe` has a height of 100% (`top: 0; bottom: 0;`) and it only contains an `iframe` as child which also has a defined height of 100%, so `.main.calypsoify.is-iframe` should never overflow.

Reply:
Hmm, with it removed I do see a double scrollbar in Chrome. I'll try landing this as is for now and we can see if we can slim this down.

Reply:
oh hmm on further refreshes it seems okay 🤔 I'll remove this in a bit